requirements.txt

```diff
@@ -4,7 +4,7 @@ filelock
 Flask
 jinja2
 jsonschema
-pydantic<2.0
+pydantic
```

Comment:
Is there a timeline for pydantic 2 on Ubuntu 22.04?  We need to consider this for this PR (note that our next release is scheduled for next month).

Reply:
Are you saying that without `pydantic<2.0` here, it will override the system pydantic version, if someone is building without a virtual env, and that is an issue?